tasks/Debian.yml

```diff
@@ -1,40 +1,59 @@
 - name: install desktop environment KDE
-  apt:  name={{ item }} state=present install_recommends=yes
-  with_items:
-    - kde-standard
-    - kde-plasma-desktop
+  apt:
+    pkg:
+      - kde-standard
+      - kde-plasma-desktop
+    state: present 
+    install_recommends: yes
   when: linux_desktop == 'kde'
+  become: yes
```

Comment:
become isn't necessary on any of these tasks because the import task has it https://github.com/pppontusw/ansible-role-linux-desktop/blob/acf6a8d8d5b24f83e99abc04e0c0701b9bb6a525/tasks/main.yml#L3